test/base-tlsutility.cpp

```diff
@@ -132,4 +150,165 @@ BOOST_AUTO_TEST_CASE(iscertuptodate_old)
 	})));
 }
 
+// openssl req -x509 -newkey ec -pkeyopt ec_paramgen_curve:secp384r1 -subj /CN=IcingaCA -days 1000000 -keyout IcingaCA.key -out IcingaCA.crt -nodes
+
+static const auto l_IcingaCACrt = R"(
+-----BEGIN CERTIFICATE-----
```

Comment:
Just my cert generator was wrong, now it works! 🎉

Reply:
Could you please elaborate what was wrong? What did you change and what caused errors?

Reply:
I don't know. I'll focus on functionality of these tests for now. This is important.